code/analytics/src/main/java/com/adobe/marketing/mobile/analytics/internal/AnalyticsHitProcessor.kt

```diff
@@ -248,6 +249,11 @@ internal class AnalyticsHitProcessor(
      */
     private fun getAnalyticsBaseUrl(state: AnalyticsState): String? {
         if (!state.isAnalyticsConfigured) {
+            Log.debug(
+                AnalyticsConstants.LOG_TAG,
+                CLASS_NAME,
+                "getAnalyticsBaseUrl - The Analytics configuration for RSID or host is not ready, as they appear to be empty or null."
```

Comment:
suggest rewording "getAnalyticsBaseUrl - The Analytics configuration for RSID or host is not found. RSID and host must not be null or empty."

Reply:
I want to keep it as debug since the other similar loggings in this file are in debug level.

Reply:
After discussion with the team, I am changing the logging for recoverable connections to Debug level.